src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -167,14 +155,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
       eventTimeAndOrderComparator
     );
 
+    const currentVenueBookMarkEvents = eventsFilledWithPriority.filter(
+      ({ isSaved, venueId: eventVenueId }) =>
+        isSaved && eventVenueId?.toLowerCase() === venueId
+    );
+
+    const currentVenueEvents = eventsFilledWithPriority.filter(
+      (event) => event.venueId === venueId?.toLowerCase()
+    );
```

Comment:
This is inconsistent. 

The original suggestion was to have a `currentVenueId` constant outside that has already been updated to lower case as to not repeat the operation and just use `venueId.toLowerCase()` inside so they're both reduced to lower before comparison

Now the namings are off, one filter function uses destructuring, the other doesn't, and there's unnecessary renaming venueId -> eventVenueId

Reply:
fix in c85b1f92ed600fc632d74bac6c9a9afb99f16203